src/binding.hpp

```diff
@@ -0,0 +1,3 @@
+#include <v8-message.h>
+
+static size_t RUST_v8__ScriptOrigin_SIZE = sizeof(v8::ScriptOrigin);
```

Comment:
A short description here and in `binding.cc` would be very helpful. I assume if a symbol is defined here it doesn't need to end up in `binding.cc`